pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/VedleggOrienteringOmRettigheter.kt

```diff
@@ -945,16 +945,22 @@ object VedleggKlagePesys : OutlinePhrase<LangBokmalNynorskEnglish>() {
                         Nynorsk to "namn, fødselsnummer og adresse",
                         English to "name, national identity number and address"
                     )
+                }
+                item {
```

Comment:
Since these are a point list, should the line before include a column at the end? "Your appeal must be made in writing and include:".

Reply:
This phrase has not been in use before, so the error has not been picked up before now.